Mathlib/Analysis/Analytic/Basic.lean

```diff
@@ -337,6 +340,98 @@ theorem radius_smul_eq (p : FormalMultilinearSeries 𝕜 E F) {c : 𝕜} (hc : c
   apply eq_of_le_of_le _ radius_le_smul
   exact radius_le_smul.trans_eq (congr_arg _ <| inv_smul_smul₀ hc p)
 
+theorem radius_compContinuousLinearMap_ge (p : FormalMultilinearSeries 𝕜 F G) (u : E →L[𝕜] F) :
+    p.radius / ‖u‖₊ ≤ (p.compContinuousLinearMap u).radius := by
+  by_cases h_zero : ‖u‖₊ = 0
+  · simp only [nnnorm_eq_zero] at h_zero
+    simp only [h_zero, nnnorm_zero, ENNReal.coe_zero]
+    by_cases hr : p.radius = 0
+    · rw [hr]
+      simp
+    · rw [ENNReal.div_zero hr, top_le_iff]
+      apply FormalMultilinearSeries.radius_eq_top_of_forall_image_add_eq_zero _ 1
+      intro m
+      ext v
+      simp only [compContinuousLinearMap_apply, ContinuousMultilinearMap.zero_apply]
+      change (p (m + 1)) 0 = 0
+      simp
```

Comment:
You should first prove this as a separate lemma:
```
p.compContinuousLinearMap (0 : E →L[𝕜] F) = constFormalMultilinearSeries 𝕜 E (p 0 0) := sorry
```
and then this step follows rather trivially.